README.md

```diff
@@ -114,6 +124,21 @@ executing the autohooks based git commit hook.
 
 Using the `poetry` mode is highly recommended.
 
+*NOTE:* If `poetry` mode is configured on an OS that has a
+`/usr/bin/env` command that does not support the `-S` split string
+option, `autohooks` will automatically change to `poetry_multiline`
+mode.
+
+### Poetry Multi-Line Mode
+
+Identical behavior to [poetry mode](#poetry-mode). By using
+`poetry__multiline` mode, the virtual environment will be activated
```

Comment:
```suggestion
`poetry_multiline` mode, the virtual environment will be activated
```

Reply:
I think I may have done the update wrong, but this was removed as an option for the user to configure.